oslabs-beta/KataLog
# Bl cf/search algo

##Overview
Issue Type

[] Bug
[x] Feature
[]Tech debt
Description
Completed search functionality to the frontend rendering of the centralized logs

Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix
Type into search to see logs that have been searched through

Previous Behavior
No searching behavior existed prior.

Expected Behavior
Logs search correctly

How Has This Been Tested
Testing on the frontend if correcting rendering of filtered logs with entered parameters occurs.

Screenshots & Videos
N/A

Additional Context
One known bug exists: when the search function is interacted with, filter functionality is lost
